mangaki/mangaki/models.py

```diff
@@ -139,6 +140,9 @@ class Work(models.Model):
     nb_dislikes = models.IntegerField(blank=True, null=False, default=0)
     controversy = models.FloatField(blank=True, null=False, default=0)
 
+    # Cache fields for the title deduplication
+    title_search = SearchVectorField('title')
```

Comment:
I think, we should track it as a further extension of the work being done in this pull request.
I'll do some exploration on it and try to determine performance, I am opening #312 for it.